pkg/controller/queue_controller.go

```diff
@@ -57,10 +57,12 @@ func (q *QueueController) Run(stopCh <-chan struct{}) {
 func (q *QueueController) runOnce() {
 	glog.V(4).Infof("Start scheduling ...")
 	defer glog.V(4).Infof("End scheduling ...")
+
 	snapshot := q.cache.Dump()
+	allPods := q.allocator.Initialize(snapshot.Pods)
```

Comment:
`Initialize` is used for policy plugin initialization, e.g. load `.so`. Prefer to add this into `Group (...)`.

Reply:
got it